texasbbq.py

```diff
@@ -265,7 +265,7 @@ def install(self, env):
             for dep in self.conda_dependencies:
                 conda_install(env, dep)
             os.chdir(self.name)
-            execute("conda run -n {} {}".format(env, self.install_command))
+            execute("conda run --no-capture-output -n {} {}".format(env, self.install_command))
```

Comment:
Is a minimum version of `conda` now required for this?

Reply:
yes, 4.9

Reply:
Would it be a good idea to enforce and/or document that?

Reply:
I am not sure where to enforce that, the dependency management for `texasbbq` is the the responsibility  of the integration tests that use it. We can document it for sure.

Reply:
Though it does download `miniconda` during operation, so I doubt this will be an issue, but for completeness, this is a good idea.